cta_indicators/README.md

```diff
@@ -10,4 +10,5 @@ This folder contains IOCs related to our publications [here](https://blogs.infob
 | 2022-06-08 | VexTrio DDGA Domains Spread Adware, Spyware, and Scam Web Forms | https://blogs.infoblox.com/cyber-threat-intelligence/cyber-threat-advisory/vextrio-ddga-domains-spread-adware-spyware-and-scam-web-forms/ | https://github.com/infobloxopen/threat-intelligence/blob/main/cta_indicators/vextrio_cta_20220606_iocs.csv |
 | 2022-08-15 | Vast Malvertising Network Hijacks Browser Settings to Spread Riskware | https://blogs.infoblox.com/cyber-threat-intelligence/cyber-threat-advisory/vast-malvertising-network-hijacks-browser-settings-to-spread-riskware/ | https://github.com/infobloxopen/threat-intelligence/blob/main/cta_indicators/omnatuor_cta_20220815_iocs.csv |
 | 2022-10-06 | Emotet: A Malware Family That Keeps Going | https://blogs.infoblox.com/cyber-threat-intelligence/emotet-a-malware-family-that-keeps-going/ | https://github.com/infobloxopen/threat-intelligence/blob/main/cta_indicators/emotet_cta_20221006_iocs.csv |
-| 2022-11-09 | Scams Using Fake Celebrity Endorsements Target EU Countries | https://blogs.infoblox.com/cyber-threat-intelligence/cyber-threat-advisory/scams-using-fake-celebrity-endorsements-target-eu-countries/ | https://github.com/infobloxopen/threat-intelligence/blob/main/cta_indicators/metacoin_cta_20221109_iocs.csv |
\ No newline at end of file
+| 2022-11-09 | Scams Using Fake Celebrity Endorsements Target EU Countries | https://blogs.infoblox.com/cyber-threat-intelligence/cyber-threat-advisory/scams-using-fake-celebrity-endorsements-target-eu-countries/ | https://github.com/infobloxopen/threat-intelligence/blob/main/cta_indicators/metacoin_cta_20221109_iocs.csv |
+2022-11-18 |  French Smishing Campaign Uses Fake Social Security Portal | https://blogs.infoblox.com/cyber-threat-intelligence/cyber-threat-advisory/french-smishing-campaign-uses-fake-social-security-portal/ | https://github.com/infobloxopen/threat-intelligence/blob/main/cta_indicators/ameli_cta_20221118_iocs.csv
```

Comment:
Are you missing the first `|` ?

Reply:
Interesting, edited it in VS code and it generated this .md file. Rendering was fine, so I looked it up. Turns out `[The pipes on either end of the table are optional](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables).`